index.html

```diff
@@ -176,7 +176,7 @@ <h2>
         and retrieve high resolution performance metric data related to the
         navigation of a document. As the <a>PerformanceNavigationTiming</a>
         interface uses [[HR-TIME-2]], all time values are measured with respect
-        to the <a data-cite="!HR-TIME-2#time-origin">time origin</a> of the
+        to the <a data-cite="HR-TIME-2#sec-time-origin">time origin</a> of the
```

Comment:
use #dfn-time-origin instead. Generally, avoid citing sections when there are links to defined concepts.

Reply:
ack